includes/class-wc-payments-checkout.php

```diff
@@ -415,16 +415,24 @@ function () use ( $prepared_customer_data ) {
 				);
 			}
 
-			// Output the form HTML.
-			if ( ! empty( $this->gateway->get_description() ) ) : ?>
-				<p><?php echo wp_kses_post( $this->gateway->get_description() ); ?></p>
+			?>
+			<div class="wcpay-upe-form" 
+				data-payment-method-type="<?php echo esc_attr( $this->gateway->get_stripe_id() ); ?>"
+				data-gateway-id="<?php echo esc_attr( $this->gateway->id ); ?>"
```

Comment:
We can do that I believe (7e7de87dd1766c2e5c7df586700402c11a117a44), although I was personally leaning toward passing it to avoid [string manipulations on the front-end](https://github.com/Automattic/woocommerce-payments/pull/9782/commits/7e7de87dd1766c2e5c7df586700402c11a117a44#diff-450d035a9e9ad47cb72bd13b3a7138e13cb0595c6a2a8c419b81524a77b10abdR134-R139). Applied the change, if you think this string manipulation is OK, let's keep this most recent change.

Reply:
Nice, thank you! There should similar references also in the tests you modified